### PR TITLE
Desktop version 2

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
created desktop version from scratch using media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,11 +13,24 @@
     <header class="header">
       <div class="logo"><a href="#logo">Artur Gouveia</a></div>
       <nav class="menu">
-        <img src="/images/Icons/Icon-Menu.png" alt="hamburguer">
```
Comment (CollinsTatang) [inline index.html:16]: - [ ] Please fix all broken image paths.

File: index.html [hunk 2]
```diff
@@ -52,6 +65,27 @@ <h3>Multi-Post Stories</h3>
         <button type="button" class="btn">See Project</button>
```
Comment (CollinsTatang) [inline index.html:65]: - [ ] Please check the See Project `button` we have it double in this section.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Arturgouveia1970, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Arturgouveia1970) [reply index.html:65]: @CollinsTatang , I do not understand this comment, I don't see how I can take it away as this button has to appear at the end of every project.
Artur

**Review by gemmen29 — APPROVED** [review]
Hi @Arturgouveia1970, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
- [ ] Kindly remove the black dots in front of social icons in the hero section to match the Figma design. you can do that by adding `list-style: none` for the ul.

| Your design | Figma design |
| --- | --- |
| image | image | 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Arturgouveia1970) [reply index.html:65]: @CollinsTatang, I still do not see it in my project. It was not in the code when I sent it. Could it be browsers issues.

**Pull request merged**